### DISCUSSION TITLE
Get metadata from video assets

### DISCUSSION THREAD

**Discussion by workatease** [directus/directus#5343] [Feature Requests]
I have been exploring the option to get metadata for video files using the [ffmpeg]( https://github.com/fluent-ffmpeg/node-fluent-ffmpeg)
but it has [prerequisite](https://github.com/fluent-ffmpeg/node-fluent-ffmpeg#prerequisites) for ffmpeg install for the environment. so there is an dependency on this library to be installed based on the OS platform and needs an environment variable set.

so my question are

- do you think we can bring this as part of directus core ?
- as it has a external library and path dependency so what do you think is the better option?

**Comment by workatease** [comment]
I have made a [video-metadata-hook](https://github.com/workatease/directus-api/tree/master/extensions/hooks/video-metadata) to update the metadata but this could be better if we have it part of directus core

**Reply by benhaynes** [reply]
As long as the dependencies aren't too crazy, then yes, I think this is valuable to have in Core!

**Reply by workatease** [reply]
Only downside I see is a performance time to generate metadata on a shared CPU

**Reply by benhaynes** [reply]
Yeah, not sure how we scope/limit that. Could have a project setting for max video size to process?

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
